engine/classes/Elgg/Database/AccessCollections.php

```diff
@@ -45,38 +109,38 @@ public function __construct($site_guid) {
 	 */
 	function getAccessList($user_guid = 0, $site_guid = 0, $flush = false) {
 		global $init_finished;
-		$cache = _elgg_services()->accessCache;
-		
+		$cache = $this->access_cache;
+
 		if ($flush) {
 			$cache->clear();
 		}
-	
+
 		if ($user_guid == 0) {
-			$user_guid = _elgg_services()->session->getLoggedInUserGuid();
+			$user_guid = $this->session->getLoggedInUserGuid();
```

Comment:
Thanks. I fixed the icon test (subtype resolution wasn't done right), so it's just the simple tests that are failing. Scrutinizer says it's a 3% increase in test coverage. I am drained, need to clear my head and rest my body.